test/e2e/testing-manifests/auth/encrypt/run-e2e.sh

```diff
@@ -136,18 +140,23 @@ main(){
     export ARTIFACTS="${ARTIFACTS:-${PWD}/_artifacts}"
     mkdir -p "${ARTIFACTS}"
 
-    export GO111MODULE=on;
-    go install sigs.k8s.io/kind@latest;
-    go install sigs.k8s.io/kubetest2@latest;
-    go install sigs.k8s.io/kubetest2/kubetest2-kind@latest;
-    go install sigs.k8s.io/kubetest2/kubetest2-tester-ginkgo@latest;
+    kube::golang::setup_env
+    (
+        # just while installing external tools
+        export GO111MODULE=on GOTOOLCHAIN=auto
+        # TODO: consider using specific versions to avoid surprise breaking changes
```

Comment:
ref: https://github.com/kubernetes/kubernetes/pull/123251#discussion_r1486890274 for why we switched to latest.

Reply:
Note: `@latest` still only has the most recent tag, which may or may not have everything for testing Kubernetes @ HEAD.

For some of the core jobs like `pull-kubernetes-e2e-kind` we're running those at HEAD of kind but I can't guarantee no changes that break every CI job, those jobs we run an extensive set of copies in the kind repo but it's already a little excessive.